storage/google/cloud/storage/blob.py

```diff
@@ -30,6 +30,9 @@
 import six
 from six.moves.urllib.parse import quote
 
+import google.auth.transport.requests as tr_requests
```

Comment:
I'd prefer if you didn't alias, maybe just `import google.auth.transport.requests` and then just use the full `google.auth.tranport.request.AuthorizedSession`?